custom_components/google_photos/camera.py

```diff
@@ -128,6 +129,22 @@ def _handle_coordinator_update(self) -> None:
             self._attr_extra_state_attributes["media_url"] = (
                 media.get("productUrl") or {}
             )
+            if media_secondary is not None:
+                self._attr_extra_state_attributes["media_secondary"] = {"media_filename":(
```

Comment:
I think it would be nicer to have 4 separate attributes,  `secondary_media_filename`,  `secondary_media_metadata`,  `secondary_media_contributor_info` and `secondary_media_url`. Reason being that not all ui components support nested attributes that well.